bindings/gumjs/gumdukvalue.c

```diff
@@ -1011,6 +1045,24 @@ _gum_duk_parse_pointer (duk_context * ctx,
     *ptr = GSIZE_TO_POINTER ((gsize) number);
     return TRUE;
   }
+  else if (gum_duk_is_uint64 (ctx, index, core))
+  {
+    guint64 val;
+
+    _gum_duk_get_uint64 (ctx, index, core, &val);
+
+    *ptr = GSIZE_TO_POINTER (val);
+    return TRUE;
+  }
+  else if (gum_duk_is_int64 (ctx, index, core))
+  {
+    gint64 val;
+
+    _gum_duk_get_int64 (ctx, index, core, &val);
+
+    *ptr = GSIZE_TO_POINTER (val);
+    return TRUE;
+  }
```

Comment:
These two should be swapped - we should check them in the same order as their chronological listing: signed then unsigned.